src/integTest/java/org/sonatype/gradle/plugins/scan/ScanPluginIntegrationTestBase.java

```diff
@@ -221,7 +253,7 @@ public void testAuditTask_IncludeTestDependencies_OssIndex() throws IOException
         .withArguments("ossIndexAudit", "--info")
```

Comment:
Both `testAuditTask_ExcludeTestDependencies_OssIndex` and `testAuditTask_IncludeTestDependencies_OssIndex` should have tests using the graph and the table to make sure we're not missing dependencies in such cases.

Reply:
> Both `testAuditTask_ExcludeTestDependencies_OssIndex` and `testAuditTask_IncludeTestDependencies_OssIndex` should have tests using the graph and the table to make sure we're not missing dependencies in such cases.

Done bac82216